evm/contracts/AccessGrants.sol

```diff
@@ -48,6 +48,7 @@ contract AccessGrants {
     function deleteGrant(
         address grantee,
         string memory dataId,
+        // UNEVIDENT why do we require the caller to know the timelock?
```

Comment:
We don't, it can be set to `"0"`. It's an argument so that Alice can specify which of the several `{Alice, Bob, 42, *}` access grants she's looking to delete

Reply:
Got it. 👌